sys/shell/commands/sc_dfplayer.c

```diff
@@ -56,7 +56,7 @@ static const char *_states[] = {
 static void _print_error(int retval)
 {
     print_str("Error: ");
-#ifdef DFPLAYER_NO_STRERROR
+#ifdef CONFIG_DFPLAYER_NO_STRERROR
```

Comment:
@maribu Does cppcheck want `if ((pos + 1) != argc)` bellow?

Reply:
It seems so :-) Feel free to change.

Reply:
Done

Reply:
Done